codewind-che-sidecar/scripts/kube/deployPFE.sh

```diff
@@ -115,6 +115,14 @@ sed -i "s/OWNER_REFERENCE_NAME_PLACEHOLDER/$OWNER_REFERENCE_NAME/g" /scripts/kub
 OWNER_REFERENCE_UID=$( kubectl get po --selector=che.original_name=che-workspace-pod,che.workspace_id=$CHE_WORKSPACE_ID -o jsonpath='{.items[0].metadata.ownerReferences[0].uid}' )
 sed -i "s/OWNER_REFERENCE_UID_PLACEHOLDER/$OWNER_REFERENCE_UID/g" /scripts/kube/codewind_template.yaml
 
+# Get the che ingress host
+CHE_INGRESS_HOST=$(kubectl get ingress -n che che-ingress -o jsonpath="{.spec.rules[0].host}")
+sed -i "s/CHE_INGRESS_HOST_PLACEHOLDER/$CHE_INGRESS_HOST/g" /scripts/kube/codewind_template.yaml
+
+# Get the che ingress domain
+CHE_INGRESS_DOMAIN=$(echo $CHE_INGRESS_HOST | sed -e "s/che-che.//g")
```

Comment:
Couple notes:
- See my other note about ingress, not sure if we need the ingress for Codewind (maybe for the performance dashboard though?)
- `che-che` won't always work for getting the ingress domain prefix. It'll depend on the name the user gave Che when installing, and which namespace Che was installed into.
- On OpenShift, Che is installed with a `Route` rather than an ingress, so this approach won't work.

Instead, I'd recommend getting the ingress domain via the `CHE_API` environment variable. It'll be something like `CHE_API=http://che-my-eclipse-che.9.37.222.5.nip.io/api`. You would just need to strip off the `/api` and `http`. You can also use the full ingress name as the prefix I believe (such as `che-my-eclipse-che.9.37.222.5.nip.io`)

Reply:
Update PR to use CHE_API instead of che-che